.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+env:
+  - PYTHON_VERSION=2.7
+before_install:
+  - wget http://repo.continuum.io/miniconda/Miniconda3-3.7.3-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda3/bin:$PATH
+  # Update conda itself
+  - conda update --yes conda
+install:
+  - conda create --yes -n env_name python=$PYTHON_VERSION pip nose numpy scipy matplotlib pandas>=0.15 flake8
+  - conda install --yes -n env_name python=$PYTHON_VERSION h5py>=2.3.1 statsmodels lxml
```

Comment:
Ok I'll make that change, thanks
On Mar 26, 2015 7:03 PM, "Yoshiki Vázquez Baeza" <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/biocore/American-Gut/pull/139#discussion_r27270223>:
>
> > @@ -0,0 +1,26 @@
> > +language: python
> > +env:
> > +  - PYTHON_VERSION=2.7
> > +before_install:
> > +  - wget http://repo.continuum.io/miniconda/Miniconda3-3.7.3-Linux-x86_64.sh -O miniconda.sh
> > +  - chmod +x miniconda.sh
> > +  - ./miniconda.sh -b
> > +  - export PATH=/home/travis/miniconda3/bin:$PATH
> > +  # Update conda itself
> > +  - conda update --yes conda
> > +install:
> > +  - conda create --yes -n env_name python=$PYTHON_VERSION pip nose numpy scipy matplotlib pandas>=0.15 flake8
> > +  - conda install --yes -n env_name python=$PYTHON_VERSION h5py>=2.3.1 statsmodels lxml
>
> I believe it will work, that's what .travis.yml in qiita does.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/American-Gut/pull/139/files#r27270223>.
>